src/canjs/controls/compare-slider.js

```diff
@@ -1,13 +1,12 @@
-import {StacheElement} from "../../can";
```

Comment:
There shouldn't be any code changes, I just formatted this file and added deprecate comment